cognite/neat/rules/models/_rules/_types/_value.py

```diff
@@ -67,42 +73,54 @@ def __str__(self) -> str:
-    {"name": "normalizedString", "python": str, "GraphQL": "String", "dms": Text},
-    {"name": "token", "python": str, "GraphQL": "String", "dms": Text},
+    {"name": "boolean", "python": bool, "GraphQL": "Boolean", "dms": Boolean, "SQL": "BOOLEAN"},
+    {"name": "float", "python": float, "GraphQL": "Float", "dms": Float64, "SQL": "FLOAT"},
+    {"name": "double", "python": float, "GraphQL": "Float", "dms": Float64, "SQL": "DOUBLE"},
+    {"name": "integer", "python": int, "GraphQL": "Int", "dms": Int32, "SQL": "INTEGER"},
+    {"name": "nonPositiveInteger", "python": int, "GraphQL": "Int", "dms": Int32, "SQL": "INTEGER"},
+    {"name": "nonNegativeInteger", "python": int, "GraphQL": "Int", "dms": Int32, "SQL": "INTEGER"},
+    {"name": "negativeInteger", "python": int, "GraphQL": "Int", "dms": Int32, "SQL": "INTEGER"},
+    {"name": "long", "python": int, "GraphQL": "Int", "dms": Int64, "SQL": "INTEGER"},
+    {"name": "string", "python": str, "GraphQL": "String", "dms": Text, "SQL": "STRING"},
+    {"name": "langString", "python": str, "GraphQL": "String", "dms": Text, "SQL": "STRING"},
+    {"name": "anyURI", "python": str, "GraphQL": "String", "dms": Text, "SQL": "STRING"},
+    {"name": "normalizedString", "python": str, "GraphQL": "String", "dms": Text, "SQL": "STRING"},
+    {"name": "token", "python": str, "GraphQL": "String", "dms": Text, "SQL": "STRING"},
     # Graphql does not have a datetime/date type this is CDF specific
-    {"name": "dateTime", "python": datetime, "GraphQL": "Timestamp", "dms": Timestamp},
-    {"name": "dateTimeStamp", "python": datetime, "GraphQL": "Timestamp", "dms": Timestamp},
-    {"name": "date", "python": date, "GraphQL": "String", "dms": Date},
+    {"name": "dateTime", "python": datetime, "GraphQL": "Timestamp", "dms": Timestamp, "SQL": "TIMESTAMP"},
+    {"name": "dateTimeStamp", "python": datetime, "GraphQL": "Timestamp", "dms": Timestamp, "SQL": "TIMESTAMP"},
+    {"name": "date", "python": date, "GraphQL": "String", "dms": Date, "SQL": "DATE"},
     # CDF specific types, not in XSD
-    {"name": "timeseries", "python": TimeSeriesReference, "GraphQL": "TimeSeries", "dms": TimeSeriesReference},
-    {"name": "file", "python": FileReference, "GraphQL": "File", "dms": FileReference},
-    {"name": "sequence", "python": SequenceReference, "GraphQL": "Sequence", "dms": SequenceReference},
-    {"name": "json", "python": Json, "GraphQL": "Json", "dms": Json},
+    {
+        "name": "timeseries",
+        "python": TimeSeriesReference,
+        "GraphQL": "TimeSeries",
+        "dms": TimeSeriesReference,
+        "SQL": "STRING",
+    },
+    {"name": "file", "python": FileReference, "GraphQL": "File", "dms": FileReference, "SQL": "STRING"},
+    {"name": "sequence", "python": SequenceReference, "GraphQL": "Sequence", "dms": SequenceReference, "SQL": "STRING"},
+    {"name": "json", "python": Json, "GraphQL": "Json", "dms": Json, "SQL": "STRING"},
 ]
 
 _DMS_TYPES: list[dict] = [
```

Comment:
I am a bit confused why there are two constants here

Reply:
prob. fastest way to make DMS complaint value types.